Desktop/html/js/table.js

```diff
@@ -801,10 +801,10 @@ JASPWidgets.tablePrimitive = JASPWidgets.View.extend({
 
 			style = JASPWidgets.Exporter.getTableContentStyles($elObj, exportParams);
 
-			if ($elObj.prop("rowspan") && $elObj.prop("rowspan") != 1)
+			/*if ($elObj.prop("rowspan") && $elObj.prop("rowspan") != 1)
 				attrs += 'rowspan="' + $elObj.prop("rowspan") + '" '
 			if ($elObj.prop("colspan") && $elObj.prop("colspan") != 1)
-				attrs += 'colspan="' + $elObj.prop("colspan") + '" '
+				attrs += 'colspan="' + $elObj.prop("colspan") + '" '*/
```

Comment:
I don't understand what it has to do with the fix... maybe could you add a comment to the uncommented stuff (or just remove it completely).

Reply:
there were double colspan entries after I removed the domparsing, but ok ill just remove it then.